cmd/nodecmd/create.go

```diff
@@ -373,9 +373,13 @@ func runAnsible(inventoryPath, avalancheGoVersion string) error {
 }
 
 func requestAWSAccountAuth() error {
-	confirm := "Do you authorize Avalanche-CLI to access your AWS account to set-up your Avalanche Validator node? " +
-		"Please note that you will be charged for AWS usage."
-	yes, err := app.Prompt.CaptureYesNo(confirm)
+	ux.Logger.PrintToUser("Do you authorize Avalanche-CLI to access your AWS account to set-up your Avalanche Validator node?")
+	ux.Logger.PrintToUser("Please note that you will be charged for AWS usage.")
+	ux.Logger.PrintToUser("By clicking yes, you are authorizing Avalanche-CLI to:")
+	ux.Logger.PrintToUser("- Set up EC2 instance and related necessary components (such as security groups, key pairs and elastic IPs)")
```

Comment:
I believe validation stuff is not done by using AWS. Are we confusing the user? He accepts here to create a node and network resources, and afterwards probably stop a node and network resources.

Reply:
would like to know @dasconnor opinion maybe we can rephrase this